resources/views/admin/dashboard.blade.php

```diff
@@ -0,0 +1,102 @@
+@extends('../layouts/main')
+@section('content-body')
+<div style="display: flex;">   
+	<!-- main side bar -->
+   <v-msidebar home="{{route('home')}}" explore="{{route('explore')}}" settings="{{route('home')}}" mycourse="{{route('my-course-review')}}" courses="{{route('courses')}}" incourse="1" active="courses"  ></v-msidebar>   
+	<!-- end main side bar -->
+
+   <div class="expr-dashboard-container" style="background-color:#eee;">
+   		<!-- top nav bar -->
+        <v-systopnave title="Dashboard" username='said'></v-systopnave>   
```

Comment:
Please update the title to My Practicals just  as it is on the design @softdiddy

Reply:
done